libs/geospatial/src/main/java/org/codice/ddf/libs/geo/util/GeospatialUtil.java

```diff
@@ -263,4 +274,66 @@ public static Geometry transformToEPSG4326LonLatFormat(
     }
     return transformedGeometry;
   }
+
+  /**
+   * Create a circular polygon from a lat, lon, radius, vertices, and distanceTolerance.
+   *
+   * @param lat - latitude in EPSG:4326 decimal degrees
+   * @param lon - longitude in EPSG:4326 decimal degrees
+   * @return radius - distance from the center point in meters
+   * @return maxVertices - the maximum number of vertices in the finished circle polygon
+   * @return distanceTolerance - the maximum distance from the original vertices a reduced vertex
+   *     may lie on a simplified circular polygon
+   */
+  public static Geometry createCirclePolygon(
+      double lat, double lon, double radius, int maxVertices, double distanceTolerance) {
+    double step = distanceTolerance;
+    Measure measure = Measure.valueOf(radius, SI.METER);
+    Point jtsPoint = new GeometryFactory().createPoint(new Coordinate(lon, lat));
+
+    Geometry bufferedCircle =
+        createBufferedCircleFromPoint(measure, DefaultGeographicCRS.WGS84, jtsPoint);
+    Geometry simplifiedCircle =
+        createBufferedCircleFromPoint(measure, DefaultGeographicCRS.WGS84, jtsPoint);
+
+    int maxVerticesWithClosedPoint = maxVertices + 1;
+
+    while (simplifiedCircle.getCoordinates().length > maxVerticesWithClosedPoint) {
+      simplifiedCircle = TopologyPreservingSimplifier.simplify(bufferedCircle, distanceTolerance);
+      distanceTolerance += step;
+    }
+
+    return simplifiedCircle;
+  }
+
+  private static Geometry createBufferedCircleFromPoint(
+      Measure<Double, Length> distance, CoordinateReferenceSystem origCRS, Geometry point) {
+    Geometry pointGeo = point;
+    MathTransform toTransform, fromTransform = null;
+
+    Unit<Length> unit = distance.getUnit();
+    if (!(origCRS instanceof ProjectedCRS)) {
+
+      double x = point.getCoordinate().x;
+      double y = point.getCoordinate().y;
+
+      String crsCode = "AUTO:42001," + x + "," + y; // CRS code for UTM
+
+      CoordinateReferenceSystem utmCrs;
+
+      try {
+        utmCrs = CRS.decode(crsCode);
+        toTransform = CRS.findMathTransform(DefaultGeographicCRS.WGS84, utmCrs);
+        fromTransform = CRS.findMathTransform(utmCrs, DefaultGeographicCRS.WGS84);
+        pointGeo = JTS.transform(point, toTransform);
+        return JTS.transform(pointGeo.buffer(distance.doubleValue(SI.METER)), fromTransform);
+      } catch (MismatchedDimensionException | TransformException | FactoryException e) {
+        LOGGER.debug("Unable to transform original CRS to UTM.", e);
```

Comment:
It might be best to add more context to this logging statement by stating that it failed to create the buffered circle from a point.